std_msgs/Header header

float64 system_time
int32   pulse_count
uint8   id